packages/salesforcedx-vscode-apex/src/commands/apexTestRunCodeAction.ts

```diff
@@ -153,7 +153,10 @@ export class ApexLibraryTestRunExecutor extends LibraryCommandletExecutor<{}> {
               .map(pkgDir => `${path.relative(workspaceFolder.uri.fsPath, pkgDir.fullPath)}/**/*.cls`)
               .flatMap(pattern => vscode.workspace.findFiles(pattern, '**/node_modules/**'))
           )
-        ).map(file => file.toString())
+        )
+          .flat()
+          // parsing to string for set to dedupe, then back to URI
+          .map(uri => uri.toString())
```

Comment:
this slipped by the compiler because without `flat` you have `URI[]` instead of `URI`

BUT array has a `toString` method so the compiler thought this was fine.